collections/_resources/responsible-computing.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Responsible Computing
+link: https://www.internetruleslab.com/responsible-computing
+skills: [TEACH,TEAM,USERS,MOD,SRU,NEW]
```

Comment:
Thank you, looks relevant!
Could you please comment on why you chose these particular skills? (this is an issue we have with all resources)

Reply:
I must admit a struggled with that. I figured the communications skills TEACH, TEAM, USERS fit quite nicely. ethical considerations are required for NEW and MOD. SRU probably requires that as well. Although that's the one I am least convinced by.

Reply:
I trust you, I just wanted to ensure a double-check.

Reply:
no worries and good call